moveit_kinematics/CMakeLists.txt

```diff
@@ -33,13 +28,25 @@ pluginlib_export_plugin_description_file(moveit_core kdl_kinematics_plugin_descr
 ament_export_libraries(${PROJECT_NAME} kdl_kinematics_plugin)
 
 include_directories(${THIS_PACKAGE_INCLUDE_DIRS}
+  ${rclcpp_INCLUDE_DIRS}
+  ${random_numbers_INCLUDE_DIRS}
+  ${moveit_msgs_INCLUDE_DIRS}
+  ${moveit_core_INCLUDE_DIRS}
+)
+
+include_directories(SYSTEM ${EIGEN3_INCLUDE_DIRS}
```

Comment:
SYSTEM ?

Reply:
Same approach as https://github.com/ros-planning/moveit2/blob/e23dc1d04d8fcabb0e561593c5316dda23c6ab5b/moveit_ros/perception/CMakeLists.txt#L85-L90.

_If the SYSTEM option is given, the compiler will be told the directories are meant as system include directories on some platforms._